webapp/src/pages/createForm/components/Question.js

```diff
@@ -0,0 +1,265 @@
+import React, { useState } from 'react';
+import { default as ReactSelect } from "react-select";
+
+const Section = ({
+    inputs, t, questions, setQuestions, num,
+    setSection, sections
+}) => {
+  const [input, setInput] = useState({
+    headline: inputs.headline,
+    placeholder: inputs.placeholder,
+    id: inputs.id,
+    order: inputs.order,
+    type: inputs.type,
+    options: inputs.options,
+    value: inputs.value,
+    label: inputs.label,
+    required: inputs.required
+  });
+
+  const handleChange = (prop) => (e) => {
+    setInput({...input, [prop]: e.target.value});
+  }
+  const handleTypeChange = (e) => {
+    setInput({...input, type: e});
+  }
+  const handleCheckChanged = (e) => {
+    setInput({...input, 'required': e.target.checked});
+  }
+
+  const handleAddOption = () => {
+    setInput({...input, options: [{
+      id: `${Math.random()}`,
+      value: input.value,
+      label: input.label,
+    }, ...input.options]});
+  }
+  const resetInputs = () => {
+    setInput({...input, value: '', label: ''});
+  }
+  const handleDeleteOption = (id, e) => {
+    const newOptions = input.options.filter(e => e.id !== id);
+    setInput({...input, options: [...newOptions]})
+  }
+
+  const updateQuestions = () => { // Update questions in the sections form when question loses focus
+    questions[num-1] = input;
+    setQuestions([...questions]);
+  }
+
+  const option = props => {
+    const { value } = props;
+    const label = <div>
+      <div className='dropdown-text'>
+        <i className={props.faClass}></i>
+      </div>
+      {t(props.label)}
+    </div>
+    return {
+      label,
+      value
+    }
+  }
+
+  const options = [
```

Comment:
Note these labels also need to be translated, you can probably just wrap `value` in `t(value)` in your option function though :)